arcflow/utils/bulk_import.py

```diff
@@ -146,13 +304,21 @@ def main():
         '--only-validate',
         action='store_true',
         help='Force only validate',)
+    parser.add_argument(
+        '--save-output-files',
+        action='store_true',
+        help='Download job output files',)
     args = parser.parse_args()
 
-    csv_bulk_import(
+    import_report = csv_bulk_import(
         csv_directory=args.dir,
         load_type=args.load_type,
         only_validate='true' if args.only_validate else 'false')
-
+    
+    if args.save_output_files:
+        retrieve_job_output(args.dir, import_report)
```

Comment:
Since `retrieve_job_output` modifies the output of `csv_bulk_import`, I recommend to put `retrieve_job_output` inside `csv_bulk_import` to maintain logical cohesion, and add an `asnake_client` parameter to `retrieve_job_output` (this also eliminates the need to instantiate an additional `ASnakeClient`).

Part 1 of 4
```suggestion
    import_report = csv_bulk_import(
            csv_directory=args.dir,
            load_type=args.load_type,
            only_validate='true' if args.only_validate else 'false',
            save_output_files=args.save_output_files)
```